src/HttpCache/composer.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "api-platform/http-cache",
+  "description": "HttpCache support",
+  "type": "library",
+  "keywords": [
+    "Cache",
+    "Http"
+  ],
+  "homepage": "https://api-platform.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Kévin Dunglas",
+      "email": "kevin@dunglas.fr",
+      "homepage": "https://dunglas.fr"
+    },
+    {
+      "name": "API Platform Community",
+      "homepage": "https://api-platform.com/comunnity/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=8.1",
+    "api-platform/metadata": "*@dev || ^3.1",
+    "api-platform/state": "*@dev || ^3.1",
+    "symfony/http-foundation": "^6.1",
+    "symfony/http-kernel": "^6.1",
+    "symfony/property-info": "^6.1",
+    "symfony/serializer": "^6.1",
+    "symfony/uid": "^6.1"
+  },
+  "require-dev": {
+    "guzzlehttp/guzzle": "^6.0 || ^7.0",
+    "symfony/dependency-injection": "^6.1",
+    "phpspec/prophecy-phpunit": "^2.0",
+    "symfony/phpunit-bridge": "^6.1",
+    "symfony/http-client": "^6.1",
+    "symfony/console": "^6.2"
```

Comment:
can you check why we have these dependencies? I'd assume that we only need guzzle, phpunit and http client. Same for the require, why the serializer / uid are needed in this component ?